resources/saas-boost-svc-onboarding.yaml

```diff
@@ -291,8 +291,7 @@ Resources:
                   - !Sub arn:aws:s3:::${SaaSBoostBucket}
               - Effect: Allow
                 Action:
-                  - s3:GetObject
-                  - s3:GetObjectVersion
+                  - s3:Get*
```

Comment:
I've spoken with Raju offline and the reason we switch to * is because we're running up against policy size limits in AWS IAM. As part of a larger story around separation into separate policies we will revisit this and get back to least privilege.